website/app/styles/components/color-swatch.scss

```diff
@@ -0,0 +1,92 @@
+.doc-color-swatch {
+  display: grid;
+  grid-template-rows: 64px 1fr;
+  width: 100%;
+  margin: 16px 0;
+  padding: 0;
+  border: 1px solid #dbdbdc;
+  border-radius: 3px;
+
+
+  @media screen and (min-width: 1280px) {
```

Comment:
After reviewing with @jorytindall, going to look into switching these to container queries instead of media queries.